src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,130 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
```

Comment:
also, it's better to put access modifiers explicitly
![image](https://github.com/mate-academy/jv-hash-map/assets/73041091/832eb701-f3eb-4bf9-aebb-9a7e36c56eaf)